lib/bundler/ruby_version.rb

```diff
@@ -64,7 +80,7 @@ def host
     #   2. ruby_version
     #   3. engine_version
     def diff(other)
-      raise ArgumentError, "Can only diff with a RubyVersion" unless other.is_a?(RubyVersion)
+      raise ArgumentError, "Can only diff with a RubyVersion, not a #{other.class}" unless other.is_a?(RubyVersion)
```

Comment:
we typically don't test the message from `ArgumentError`s and the like, since they're mor a debugging tool than anything else